.moai/specs/SPEC-ENVKEY-ANTHROPIC-SSOT-001/spec.md

```
@@ -0,0 +1,377 @@
+| `internal/hook/session_start.go` | 8 |
+| `internal/cli/launcher.go` | 7 |
+| `internal/cli/settings.go` | 6 |
+| `internal/tmux/cg_detect.go` | 4 |
+| `internal/statusline/metrics.go` | 3 |
+| `internal/sandbox/env.go` | 1 |
+| `internal/cli/worktree/tmux_integration.go` | 1 |
+| **Total** | **83** |
+
+Nine distinct literal values are in use:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)